### PR TITLE
Fix for Creators Updaters

### PR DESCRIPTION
**Pull request by elizabeth-jimenez**

### CODE REVIEW

File: talentmap_api/fsbid/services/agenda.py [hunk 1]
```diff
@@ -236,14 +236,11 @@ def fsbid_single_agenda_item_to_talentmap_single_agenda_item(data, remarks={}):
     statusFull = data.get("aisdesctext", None)
     updaters = pydash.get(data, "updaters") or None
     if updaters:
-        updaters = (list(map(
-            fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters,
-            pydash.get(data, "updaters")
-        )))[0]
+        updaters = fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters(updaters[0])
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda.py:239]: updated mock data and verified the data is coming through correctly

File: talentmap_api/fsbid/services/agenda.py [hunk 2]
```diff
@@ -331,8 +328,6 @@ def map_tf(tf=None):
     return res
 
 def fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters(data):
-    if not data:
-        return {}
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda.py:335]: 🦅🧼

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez any other testing expected for this besides verifying the data was coming through as expected?

File: talentmap_api/fsbid/services/agenda.py [hunk 3]
```diff
@@ -236,14 +236,11 @@ def fsbid_single_agenda_item_to_talentmap_single_agenda_item(data, remarks={}):
...
-        updaters = (list(map(
-            fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters,
-            pydash.get(data, "updaters")
-        )))[0]
+        updaters = fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters(updaters[0])
 
     creators = pydash.get(data, "creators") or None
     if creators:
-        fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters(creators[0])
+        creators = fsbid_ai_creators_updaters_to_talentmap_ai_creators_updaters(creators[0])
```
Comment (SidelineCory24) [inline talentmap_api/fsbid/services/agenda.py:243]: when trying to print out `updaters[0]` or `creators[0]` after the function call I was getting error. Is that behavior expected?

**Comment by scott062** [comment]
@SidelineCory24 @elizabeth-jimenez How close are we to merging this?

**Comment by SidelineCory24** [comment]
> @SidelineCory24 @elizabeth-jimenez How close are we to merging this?

Around 80% maybe (it's beens some time lol). There's some troubleshooting that still needs to be done and I sent over my testing steps right before Elizabeth's end of year leave started.

**Pull request merged**